enhancements/cel-scanner.md

```diff
@@ -0,0 +1,508 @@
+# CEL-Based Scanner for Compliance Operator (TechPreview)
+
+## Summary
+
+This enhancement proposes introducing a **new CRD** called `CustomRule` to provide **CEL** (Common Expression Language) scanning capabilities in the Compliance Operator. By creating a stand-alone CRD, instead of modifying the existing `ComplianceScan` resource, we preserve backward compatibility for the OpenSCAP-based workflows.
```

Comment:
This also means we have the ability to experiment with the CEL scanning implementation in `CustomRule` before we bake it into the `Rule`.